pkg/querier/queryrange/roundtrip.go

```diff
@@ -232,7 +232,7 @@ func getOperation(path string) string {
 		return QueryRangeOp
 	case strings.HasSuffix(path, "/series"):
 		return SeriesOp
-	case strings.HasSuffix(path, "/labels") || strings.HasSuffix(path, "/label"):
+	case strings.HasSuffix(path, "/labels") || strings.HasSuffix(path, "/label") || strings.HasSuffix(path, "/values"):
```

Comment:
This function feels a bit flimsy, and could introduce some _interesting_ bugs in the future.
We should refactor this at some point

Reply:
Thanks for adding a test!